### PR TITLE
Support for Tipsy parameter on ingest

### PR DESCRIPTION
**Pull request by BrandonHaynes**

### CODE REVIEW

**Comment by senderista** [comment]
@BrandonHaynes not sure if these TypeErrors in the tests are new, but they need to be fixed.

**Comment by BrandonHaynes** [comment]
@bmyerz Any idea why these tests started failing?  I don't think they're related to this PR.

**Comment by bmyerz** [comment]
@BrandonHaynes checking it out. I'm wondering if something about travis changed. These are new tests but they've passed in master when they got merged.

**Comment by bmyerz** [comment]
appears to be due to Travis #493

**Comment by BrandonHaynes** [comment]
@bmyerz @senderista I went ahead and corrected the change in behavior with httmock in this PR; should be good to merge.

**Comment by bmyerz** [comment]
@BrandonHaynes find a workaround at #493

**Comment by BrandonHaynes** [comment]
@bmyerz While #493 contains a workaround, this PR now contains an actual fix for the httmock issue :)

**Comment by bmyerz** [comment]
@BrandonHaynes awesome! I figured later httmock version was more careful about the spec

File: raco/myrial/parser.py [hunk 1]
```diff
@@ -611,11 +611,15 @@ def p_file_parser_fun(p):
...
- | OPP LPAREN RPAREN"""
+ | OPP LPAREN RPAREN
+ | TIPSY LPAREN option_list RPAREN
+ | TIPSY"""
         if len(p) == 7:
             format, schema, options = (p[1], p[3], dict(p[5]))
         elif len(p) == 5:
-            format, schema, options = (p[1], p[3], {})
+            format = p[1]
+            schema = p[3] if p[1] != 'TIPSY' else []
```
Comment (senderista) [inline raco/myrial/parser.py]: Can we add a TODO comment to split this into separate productions for each format?

File: raco/backends/myria/myria.py [hunk 2]
```diff
@@ -191,6 +191,15 @@ def compileme(self):
         }, **self.options)
         if self.format == 'OPP':
             encoding['opType'] = "SeaFlowScan"
+        elif self.format == 'TIPSY':
```
Comment (senderista) [inline raco/backends/myria/myria.py]: I think you might as well just overwrite `encoding` with another assignment rather than delete attrs etc.

**Comment by bmyerz** [comment]
You can find the fix for the above failed test at https://github.com/uwescience/raco/tree/tipsy-fix-get-sql

**Comment by senderista** [comment]
`tipsy` branch merged in 2d8af658492e6424020ea60d44c877829a2cf254

**Pull request closed without merging**